formats/json-tests/jvmTest/src/kotlinx/serialization/JvmMissingFieldsExceptionTest.kt

```diff
@@ -132,10 +129,10 @@ class JvmMissingFieldsExceptionTest {
     }
 
 
-    private inline fun assertFailsWithMessages(messages: List<String>, block: () -> Unit) {
-        val exception = assertFailsWith(SerializationException::class, null, block)
-        assertEquals("kotlinx.serialization.MissingFieldException", exception::class.qualifiedName)
-        val missedMessages = messages.filter { !exception.message!!.contains(it) }
+    private inline fun assertFailsWithMessages(fields: List<String>, block: () -> Unit) {
+        val exception = assertFailsWith(MissingFieldException::class, null, block)
+        val missedMessages = fields.filter { !exception.message!!.contains(it) }
```

Comment:
M.b. `missedFields`?
Such a check is not very reliable `!exception.message!!.contains(it)`, maybe always put names in single quotes and check `!exception.message!!.contains("'$it'")`